setup.py

```diff
@@ -112,15 +110,12 @@
     'Source': 'https://github.com/theupdateframework/tuf',
     'Issues': 'https://github.com/theupdateframework/tuf/issues'
   },
-  python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4",
```

Comment:
Do we still want the `<4`?

Reply:
Good question! We retain this behaviour with `python_requires ="~=3.6"`, because the ~= clause indicates compatible releases. As 4 would be a major version change, that will not be compatible with 3.6.

See [[1](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires)] and [[2]](https://www.python.org/dev/peps/pep-0440/#version-specifiers)